docs/running from source.md

```diff
@@ -0,0 +1,53 @@
+# Running From Source
+
+If you just want to play and there is a compiled version available on the
```

Comment:
I think the "if there's a compiled version" can be removed here since that is no longer an issue in our current environment and just mention downloading the latest release.

Reply:
we don't have mac builds yet :/